### ISSUE TITLE
No forecasts for Lausanne

### ISSUE THREAD

**Issue by pherjung** [ichthyosaurus/harbour-meteoswiss#3]
If I add Lausanne (I didn't try all variant), a blank "line" is showed with the city name, but nothing else. If I click on it, data is beeing loaded and only sun times is showed.

I'm using version 1.3.1 on a Xperia 10 II with Sailfish 4.4

![Capture d'écran_20220330_001](https://user-images.githubusercontent.com/5952038/160728720-06edda6e-1e08-400c-a62a-04ff20107b04.png)

**Comment by ichthyosaurus** [comment]
I quickly checked it: 1001 fails, 1004 works. I'll investigate further but at least you could use another zip code from the area for now...

**Comment by pherjung** [comment]
Just launched from terminal and I get following error :
```
[W] unknown:191 - file///usr/share/harbour-meteoswiss/qml/js/forecast.js:191: TypeError: Cannot read property 'precipitation10m' of undefined
```

**Comment by ichthyosaurus** [comment]
Yes, I saw that too. I'm currently thinking about maybe ignoring this bug for the time being, until I find the time to rewrite the relevant code. (I wanted to port the main code from JS to Python since the beginning...)

Actually, there is no local forecast available for both 1001 and 1002. They are also not mentioned on https://www.cadastre.ch/de/services/service/registry/plz.html , even though Lausanne uses 1000-1018 according to [Wikipedia](https://de.wikipedia.org/wiki/Lausanne). Do you happen to know if 1000 / 1001 / 1002 are actually in use?

**Comment by pherjung** [comment]
Well, no stress. There is a workaround and it's okay.

**Comment by ichthyosaurus** [comment]
Roughly 200 zip codes (including 1000, 1001, 1002) are not supported (anymore) by the official app, and this app uses that app's data. In v2.0.0 I added a notification when you're trying to refresh an unsupported location. You can still add those locations through the search because I didn't remove the broken locations from the search database yet.

You can simply add 1003 and everything should work fine.
